doc/releasenotes/17_0_0_summary.md

```diff
@@ -168,4 +185,13 @@ Some notes to help understand these metrics:
 
 ## <a id="deprecations-and-deletions"/> Deprecations and Deletions
 
-* The deprecated `automation` and `automationservice` protobuf definitions and associated client and server packages have been removed.
\ No newline at end of file
+* The deprecated `automation` and `automationservice` protobuf definitions and associated client and server packages have been removed.
+
+### <a id="deprecated-stats"/>Deprecated Stats
+
+These stats are deprecated in v17.
+
+| Deprecated stat | Supported alternatives |
+|-|-|
+| `backup_duration_seconds` | `BackupDurationNanoseconds` |
+| `restore_duration_seconds` | `RestoreDurationNanoseconds` |
```

Comment:
Was this missed in a previous PR? Am wondering because it is unrelated to the rest of this PR.

Reply:
No they weren't missed, but I moved them further down in the release notes because it seems more appropriate. Can revert if you prefer.